A32NX/html_ui/Pages/A32NX_Core/A32NX_GPWS.js

```diff
@@ -25,18 +26,60 @@ class A32NX_GPWS {
         const altitude = Simplane.getAltitudeAboveGround();
         const vSpeed = Simplane.getVerticalSpeed();
         const Airspeed = SimVar.GetSimVarValue("AIRSPEED INDICATED", "Knots");
+        const mda = SimVar.GetSimVarValue("L:AIRLINER_MINIMUM_DESCENT_ALTITUDE", "Number");
+        const dh = SimVar.GetSimVarValue("L:AIRLINER_DECISION_HEIGHT", "Number");
+        const phase = SimVar.GetSimVarValue("L:AIRLINER_FLIGHT_PHASE", "Enum");
 
         if (!SYS_PushButton && altitude >= 10 && altitude <= 2450) { //Activate between 10 - 2450 radio alt unless SYS is off
             this.gpws_pull_up(altitude, vSpeed);
             this.gpws_sink_rate(altitude, vSpeed);
-            this.gpws_dont_sink(altitude, vSpeed);
-            this.gpws_too_low(altitude, Airspeed, FLAP_PushButton);
+            this.gpws_dont_sink(altitude, vSpeed, phase);
+            this.gpws_too_low(altitude, Airspeed, FLAP_PushButton, phase);
         } else {
             SimVar.SetSimVarValue("L:GPWS_SINK_RATE", "Bool", 0);
             SimVar.SetSimVarValue("L:GPWS_PULL_UP", "Bool", 0);
             SimVar.SetSimVarValue("L:GPWS_DONT_SINK", "Bool", 0);
             SimVar.SetSimVarValue("L:GPWS_TOO_LOW", "Enum", 0);
         }
+
+        if ((mda !== 0 || dh !== -1) && phase === FlightPhase.FLIGHT_PHASE_APPROACH) {
+            let minimumsDA; //MDA or DH
+            let minimumsIA; //radio or baro altitude
+            const radioAlt = SimVar.GetSimVarValue("PLANE ALT ABOVE GROUND MINUS CG", "Feet");
+            const baroAlt = SimVar.GetSimVarValue("INDICATED ALTITUDE", "feet");
+            if (dh > -1) {
```

Comment:
`if(dh >= 0)` will be better no ? (-1 is the default value but negative DH is not allowed)

Reply:
Else LGTM, good work